### DISCUSSION TITLE
github copilot

### DISCUSSION THREAD

**Discussion by Stulie17Sampindi** [community/community#106902] [GitHub Education]
### Select Topic Area

General

### Body

i am trying to use my github copilot chat but its taking me to free trial but i am on dtudent developer pack and my subscription is active. what may be the problem

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Stulie17Sampindi , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by Stulie17Sampindi** [reply]
thank you
